tools/how-the-browser-creates-pages/index.md

```diff
@@ -251,6 +251,41 @@ _Композитинг_ — это разделение содержимого
 
 </aside>
 
+Иногда анимация, применённая к элементу, не вызывает выделение в отдельный слой, потому что не распознаётся браузером как триггер для композитинга. Однако мы всё равно хотим стриггерить композитинг, чтобы добиться плавности. Для этого есть несколько решений.
+
+Во-первых, можно воспользоваться хаком и вызвать на элементе пустую анимацию, являющуюся триггером. Например:
```

Comment:
Не уверена что tranlsateZ вызывает какую-либо анимацию. Можешь поделиться ссылочкой на эту инфу?

Reply:
Гугление привело к такому выводу:

> And specifically `translateZ(0)` will create a new layer, which forces the browser to shift the rendering to the GPU rather than the CPU, ie force CSS animations will use the graphics hardware on a device rather than the main processor

Но есть [какая-то бага в Safari](https://blog.frobieter.de/fix-safari-transform-z-index-bug/).

Reply:
О, это очень популярное заблуждение :)  
And specifically translateZ(0) will create a new layer, which forces the browser to shift the rendering to the GPU rather than the CPU, ie force CSS animations will use the graphics hardware on a device rather than the main processor

На самом деле браузер _всегда_ использует GPU, но с транслейтами он делает это немножк иначе

Reply:
Оаааааа.